homeassistant/components/ovo_energy/__init__.py

```diff
@@ -0,0 +1,112 @@
+"""Support for OVO Energy."""
+import logging
+from typing import Any, Dict
+
+import aiohttp
+from ovoenergy.ovoenergy import OVOEnergy
+
+from homeassistant.components.ovo_energy.const import (
+    CONF_ACCOUNT_ID,
+    DATA_OVO_ENERGY_CLIENT,
+    DOMAIN,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.typing import ConfigType, HomeAssistantType
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup(hass: HomeAssistantType, config: ConfigType) -> bool:
+    """Set up the OVO Energy components."""
+    return True
```

Comment:
If this isn't needed, just remove it?

Reply:
I get this error if I remove this function:

```bash
2020-06-21 15:03:13 ERROR (MainThread) [homeassistant.setup] Setup failed for ovo_energy: No setup function defined.
```